libs/evm-tools/src/selectors/index.ts

```diff
@@ -24,8 +40,25 @@ export enum ERC1155_CLEAR_SIGNED_SELECTORS {
   SAFE_BATCH_TRANSFER_FROM = "0x2eb2c2d6",
 }
 
+export const DAPP_SELECTORS: Record<string, string> = {
+  ...ARBITRUM_CLEAR_SIGNED_SELECTORS,
+  ...AVALANCHE_C_CHAIN_CLEAR_SIGNED_SELECTORS,
+  ...BASE_CLEAR_SIGNED_SELECTORS,
+  ...BSC_CLEAR_SIGNED_SELECTORS,
+  ...ETHEREUM_CLEAR_SIGNED_SELECTORS,
+  ...ETHEREUM_GOERLI_CLEAR_SIGNED_SELECTORS,
+  ...ETHEREUM_HOLESKY_CLEAR_SIGNED_SELECTORS,
+  ...ETHEREUM_ROPSTEN_CLEAR_SIGNED_SELECTORS,
+  ...FANTOM_CLEAR_SIGNED_SELECTORS,
+  ...OPTIMISM_CLEAR_SIGNED_SELECTORS,
+  ...POLYGON_CLEAR_SIGNED_SELECTORS,
+  ...POLYGON_ZK_EVM_CLEAR_SIGNED_SELECTORS,
+  ...ZKSYNC_CLEAR_SIGNED_SELECTORS,
+};
```

Comment:
We could probably export this directly in the other file but not a big deal to keep it here too